util/clean_up.py

```diff
@@ -0,0 +1,19 @@
+from submodules.model.business_objects import upload_task
+import os, shutil
+
+def clean_up_database() -> None:
+    upload_task.remove_all_keys(with_commit=True)
+
+def clean_up_disk() -> None:
```

Comment:
Do i understand it right das clean_up_disk only checks the tmp folder? Maybe we should add this in the method name since I first thought I need to worry about all files that could be erased :D

Reply:
Exactly, it checks and cleans up only the tmp folder. I wrote it with this name so if we want to have further clean up steps they could just be integrated in this method, but of course I can add something like "tmp" in the name of the method